cfgov/locale/es/LC_MESSAGES/django.po

```diff
@@ -224,6 +228,10 @@ msgstr "/es/obtener-respuestas/buscar/"
 msgid "Search for your question"
 msgstr "Busca su pregunta"
 
+#: jinja2/v1/_includes/organisms/ask-search.html:63
+msgid "Search Ask CFPB for your question"
+msgstr "Busque Pregunte a CFPB por su pregunta"
```

Comment:
@OrlandoSoto is this translation okay?

Reply:
or would @doguin mind reviewing these Spanish translations?

Reply:
Ask CFPB in Spanish is translated as "Obtener Respuestas" (https://www.consumerfinance.gov/es/obtener-respuestas/), so maybe we should use that instead of "Pregunte a CFPB"

Reply:
> or would @doguin mind reviewing these Spanish translations?

"Search for you question" should be "Busque su pregunta". Not, "Busca su pregunta"

Reply:
> Ask CFPB in Spanish is translated as "Obtener Respuestas" (https://www.consumerfinance.gov/es/obtener-respuestas/), so maybe we should use that instead of "Pregunte a CFPB"

@sonnakim Agreed - I think it should be "Busque 'Obtener Respuestas' para su cuestionamiento"

Reply:
Updated here https://github.com/cfpb/consumerfinance.gov/pull/6493/commits/4482e0ab523f8f1279ffb38b862e96890ad3f714